### ISSUE TITLE
 Enable SSL for this domain / Use Lets Encrypt to obtain SSL certificate / Error code:15

### ISSUE THREAD

**Issue by Johnzapf** [hestiacp/hestiacp#1373]
I just setup a new install of Hestia on Debian 10. Everything went well and all is working except for:  Enable SSL for this domain / Use Lets Encrypt to obtain SSL certificate, getting Error code 15. 

And when I go to the new site URL its forcing me to use https and that's not checked. 

I have deleted the domain and re added and I am getting the same error.  I would love to use Hestia and the auto/free SSL is a huge part of my decision to use this, I really hope I can get it to work. FYI it works fine in VestaCP so I am not sure why its not working with Hestia.

Thanks, John

**Comment by Johnzapf** [comment]
Also I tried this:

v-change-sys-hostname host.domain.tld

v-add-letsencrypt-host

and now I get  -  Error: Let's Encrypt new auth status 429

**Comment by jaapmarcus** [comment]
Please check https://community.letsencrypt.org/t/error-429-too-many-requests/98580

**Comment by jaapmarcus** [comment]
Also please use our support forum (https://forum.hestiacp.com) or Discord for Technical support. 

429 are "Rate limit request for Lets encrypt" wait about 1 hour and try again. 

Error code 15. DNS is not set up properly for your hostname.

**Comment by Johnzapf** [comment]
so weird. I jsut redid all again and now still getting Error: Let's Encrypt new auth status 429 every time.

DNS is fine this works perfect with VestaCP. I have looked at the like you sent before I came to the forum. let me know when you have some ideas on what is going on here.

and now I get this everytime I click on most things in the admin pannel:
Your connection isn't private
Attackers might be trying to steal your information from host.website.com (for example, passwords, messages, or credit cards).
NET::ERR_CERT_AUTHORITY_INVALID

and the main admin sites SSL show green so that one should be fine.

Thanks John
